template.distillery/Makefile.style

```diff
@@ -18,14 +18,23 @@ define ERROR_SASS
 
 Error: SASS not found.
 
-Ocsigen-start gives the choice to use SASS (a CSS preprocessor) to write
+Ocsigen Start gives the choice to use SASS (a CSS preprocessor) to write
```

Comment:
Yes. The name of the project is Ocsigen Start.
Like Google Search, or Microsoft Word.